### PR TITLE
feat: new index pages, data. and form

### PR DESCRIPTION
**Pull request by putradnata**
- Add: Airport Index Page
- Add: Users Index Page
- Add: Car Class Index Page
- Add: Airport Create Page (including form)
- Add: Icons
- Add: Some placeholders data for Airport, Car Classes, and Users
- Update: Sidebar Menu Items

### CODE REVIEW

File: pages/admin/airport/[id].tsx
Comment (permanar) [inline pages/admin/airport/[id].tsx:1]: While it's not really recommended approach, you can't leave an empty file like this.

Please at least do the bare minimum to make this page works (i.e.: add export default and return empty JSX).

File: pages/admin/airport/create.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import Link from 'next/link';
+
+export default function CreateAirport() {
+    const dispatch = useDispatch();
+
+    const initialValues = {
+        name: '',
+        latitude: '',
+        longitude: '',
+    };
```
Comment (permanar) [inline pages/admin/airport/create.tsx:15]: Please move this code outside the `CreateAirport()` function (at root level).

File: pages/admin/carclass/[id].tsx
Comment (permanar) [inline pages/admin/carclass/[id].tsx:1]: Please do the same as the airport page above.

File: pages/admin/carclass/create.tsx
Comment (permanar) [inline pages/admin/carclass/create.tsx:1]: Please do the same as the airport page above.

File: pages/admin/orders/[id].tsx
Comment (permanar) [inline pages/admin/orders/[id].tsx:1]: Please do the same as the airport page above.

File: pages/admin/orders/index.tsx
Comment (permanar) [inline pages/admin/orders/index.tsx:1]: Please do the same as the airport page above.

File: pages/admin/users/[id].tsx
Comment (permanar) [inline pages/admin/users/[id].tsx:1]: Please do the same as the airport page above.

File: pages/admin/users/create.tsx
Comment (permanar) [inline pages/admin/users/create.tsx:1]: Please do the same as the airport page above.

**Review by permanar — CHANGES_REQUESTED** [review]

**Comment by putradnata** [comment]
Resolve #17

**Pull request closed without merging**